### PR TITLE
Add mpirun for steps 70 & 75 output files

### PR DESCRIPTION
**Pull request by jpthiele**
In the hope of fixing #17263 I added `mpirun=1` to the output files for step 70 and 75, 
even though locally on my laptop both tests passed without.

I also copied the output file for step-70 to have one each for `with_petsc=true` and `with-trilinos=true`
since step-70 should compile and run successfully with both libraries.

### CODE REVIEW

**Comment by jpthiele** [comment]
I think both Trilinos and PETSc require deal.II to be configured with MPI so it might be superflous,
but it certainly would not hurt.

File: tests/examples/step-70.mpirun=1.with_petsc=true.with_p4est=true.output
Comment (masterleinad) [inline tests/examples/step-70.mpirun=1.with_petsc=true.with_p4est=true.output:1]: It seems `trilinos` is the default so would it make more sense to have one variant for `petsc=on, trilinos=off` and one for `trilinos=on`?

Comment (jpthiele) [reply tests/examples/step-70.mpirun=1.with_petsc=true.with_p4est=true.output:1]: Ah yes, but the general question would be whether or not there is a configuration like that on the test servers?

**Comment by jpthiele** [comment]
> On my laptop, I can only have tests like step-70*.output with different numbers of mpi ranks. I think this comes from the structure of build/tests/examples/binary/step-70 with subdirectories /serial, /mpirun=1, /mpirun=2, /mpirun=3 etc. So you can't have two .output file with mpirun=1.

I see the debug parallel action also failed for that same reason. Then we should just keep that as is.

File: tests/examples/step-70.mpirun=1.with_petsc=true.with_p4est=true.output
Comment (jpthiele) [reply tests/examples/step-70.mpirun=1.with_petsc=true.with_p4est=true.output:1]: Also, the way the tests are set up currently this is not allowed anyways, unless it is possible to switch the order and put mpirun further down the line.

Comment (masterleinad) [reply tests/examples/step-70.mpirun=1.with_petsc=true.with_p4est=true.output:1]: Not necessarily. Developers might also have that configuration and want the tests to work.

Comment (masterleinad) [reply tests/examples/step-70.mpirun=1.with_petsc=true.with_p4est=true.output:1]: I mean it's fair just not to provide a variant for now.

**Comment by jpthiele** [comment]
I think switching around to have `with_trilinos=true|false` in the beginning made the most sense and should work from the documentation of the test suite.

File: tests/examples/step-70.with_trilinos=false.with_petsc=true.with_p4est=true.mpirun=1.output
Comment (masterleinad) [inline tests/examples/step-70.with_trilinos=false.with_petsc=true.with_p4est=true.mpirun=1.output:1]: This file has
```
Running StokesImmersedProblem<2> using Trilinos.
```
which needs to be adapted when this is supposed to run using PETSc.

Comment (jpthiele) [reply tests/examples/step-70.with_trilinos=false.with_petsc=true.with_p4est=true.mpirun=1.output:1]: yes, right. Changed it

**Review by masterleinad — APPROVED** [review]

**Review by tamiko — APPROVED** [review]

**Comment by tamiko** [comment]
@jpthiele `output` file variants for tests are quite a bit quirky. For a single test there must always be at most one valid output file. This means it is important that constraints for different output files (but for the same test) are never true at the same time. This is the case now as we have one variant with `with_trilinos=true` and another one with `with_trilinos=false`.

**Pull request merged**